fedora-manual-setup.md

```diff
@@ -58,7 +58,7 @@ sudo systemctl restart redis
 
 ### Rails cache (Memcached)
 
-If available, Redis will use [Memcached](https://www.memcached.org) for caching. Installing it is completely optional but still recommended. We recommend running memcached in a [Podman](https://podman.io/)~~~~ container:
+If available, Redis will use [Memcached](https://www.memcached.org) for caching. Installing it is completely optional but still recommended. We recommend running memcached in a [Podman](https://podman.io/) container:
```

Comment:
it is not only Redis that uses memcached. Maybe here is the place to note how to enable memcached in rails for whoever wanted. I remembere us discussing how to enable it and how to document or to enable unconditionally.

But pull request is an improvement already. So can be merged without further changes.

Reply:
done